data/scenariolens-avd.data.json

```
@@ -0,0 +1,184 @@
+[
+  {
+    "type": "Design Considerations",
+    "pillars": [
+      "scenario/AVD"
+    ],
+    "lens": "scenario",
+    "category": "AVD",
+    "subCategory": "FSLogix",
+    "title": "[FSLogix](https://docs.microsoft.com/en-us/azure/architecture/example-scenario/wvd/windows-virtual-desktop-fslogix) technology allows for users profiles to be stored in a central location and accessed by AVD Session Hosts, side loading the disks in a seamless manner. If using a solution where your users can sign into different Azure Virtual Desktop Session Hosts (Multi-session for example), [FSLogix](https://docs.microsoft.com/azure/architecture/example-scenario/wvd/windows-virtual-desktop-fslogix) should be used to centralize users' profiles.",
+    "recommendation": {
+      "title": "Review your central profile store architecture and incorporate FSLogix if you are using multi-session",
+      "context": "",
+      "links": []
+    },
+    "children": [
+      {
+        "title": "(Storage) Storage requirements should be optimized for performance in the Azure Virtual Desktop service",
```

```suggestion
        "title": "Storage requirements should be optimized for performance in the Azure Virtual Desktop service",
```

I think these are unnecessary.